ec2utils/ec2deprecateimg/lib/ec2utils/ec2DepImgExceptions.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2015 SUSE LLC, Robert Schweikert <rjschwei@suse.com>
+#
+# This file is part of ec2deprecateimg.
+#
+# ec2deprecateimg is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ec2deprecateimg is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ec2deprecateimg.  If not, see <http://www.gnu.org/licenses/>.
+
+class EC2DeprecateImgException(Exception):
```

Comment:
To allow us to raise and catch a custom exception.

Looking ahead I think as more utilities are developed we might want to integrate all the command lines into something like:

ec2utils depreceteimage

and collect all the exceptions into 1 file ec2utilsExceptions.py > We would not want to raise generic exceptions everywhere, that would be confusing.